modules/passkey/test/utils/safe.ts

```diff
@@ -0,0 +1,75 @@
+import { Hex } from '@noble/curves/abstract/utils'
+import { ethers } from 'hardhat'
+import { Address } from 'hardhat-deploy/types'
```

Comment:
```suggestion
import { AddressLike, BytesLike } from 'ethers'
import { ethers } from 'hardhat'
```

Reply:
`AddressLike` cannot be used for `verifyingContract`, so using `Address` for that, rest changed 👍🏾